charts/substra-backend/templates/job-image-builder-warmer.yaml

```diff
@@ -130,20 +161,52 @@ spec:
                 values:
                 - {{ template "substra.name" $ }}-worker
             topologyKey: kubernetes.io/hostname
-      containers:
-      - name: makisu-cache-warmer
+      initContainers:
+      - name: populate-warmup-folder
         image: gcr.io/uber-container-tools/makisu:v0.2.0
         args:
           - "build"
           - "-t={{ $img }}:warm"
+          - "--modifyfs=true"
           - "/makisu-context"
         volumeMounts:
         - name: makisu-cache
           mountPath: /makisu-storage
+          subPath: warm-{{ $index }}
           readOnly: False
         - name: dockerfile
           mountPath: /makisu-context
           readOnly: true
+      - name: copy-to-cache
```

Comment:
Note: this won’t  work if we preload mutiple images. 

To fix it: Instead of adding multiple jobs, add multiple “copy-to-cache” init containers run sequentially, all pointing to the same cache folder (no subpath)

Reply:
Is it still an issue ?